csharp-intermediate-topics/ConstVsReadonlyInCSharp/ConstVsReadonlyInCSharp/TaxCalculator.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ConstVsReadonlyInCSharp
+{
+    public class TaxCalculator
+    {
+        private readonly double _countryVAT;
+        private readonly double _euroToDollarConversionRate = 1.08;
+        private readonly MathCalculator _calculator = new MathCalculator();
```

Comment:
By making the `MathCalculator` class static, this line can be removed and you can just use the static methods where necessary in the code

Reply:
I use it as an example of how to assign a complex type to a readonly field

Reply:
Missed that being the use case.